### PR TITLE
Add url-search-params@0.7.0 w/ git auto-update

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #10614
@cdnjs/intern3 Please help me review this pull request, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/WebReflection/url-search-params
 * NPM package url (optional): https://www.npmjs.com/package/url-search-params
 * License and its reference: [MIT](https://github.com/WebReflection/url-search-params/blob/master/LICENSE.txt)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 5
   - Count of stars: 237
   - Count of forks: 21
 * NPM download stats (optional):
   - Downloads in the last day: 1,600
   - Downloads in the last week: 7,503
   - Downloads in the last month: 32,988

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/url-search-params/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  "description": "Simple polyfill for URLSearchParams standard",
+  "keywords": [
+    "DOM",
+    "URLSearchParams",
+    "polyfill",
+    "shim"
+  ],
+  "author": {
+    "name": "Andrea Giammarchi",
+    "web": "http://webreflection.blogspot.com/"
```
Comment (hare1039) [inline ajax/libs/url-search-params/package.json]: I think `https` is a better one.

**Comment by extend1994** [comment]
@hare1039 @clairetsai818 I modified the author info. Please help me review it, thanks.

File: ajax/libs/url-search-params/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    "url": "git://github.com/WebReflection/url-search-params.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/WebReflection/url-search-params.git",
+    "fileMap": [
+      {
+        "basePath": "build",
+        "files": [
+          "url-search-params.js"
```
Comment (pvnr0082t) [inline ajax/libs/url-search-params/package.json]: @extend1994 How about other files under `build` directory? Could you confirm that if those files need to be hosted in cdnjs. Thanks.

Comment (extend1994) [inline ajax/libs/url-search-params/package.json]: @pvnr0082t I just opened an issue about it. Once the author reply, I will report here.

Comment (extend1994) [inline ajax/libs/url-search-params/package.json]: @pvnr0082t The author has replied. I think `url-search-params.js`(minified one) and `url-search-params.max.js`(not minified one) are what we need. Could you take a look at the author's reply to make sure if my understanding is correct. Here it is: WebReflection/url-search-params#18

Comment (pvnr0082t) [inline ajax/libs/url-search-params/package.json]: @extend1994 It seems that `url-search-params.node.js` is the only one we don't need.

**Comment by extend1994** [comment]
@pvnr0082t Thank you for double check and please help me review it, thanks.

**Comment by extend1994** [comment]
@pvnr0082t I removed spare files, please help me review it, thanks!

**Pull request merged**